src/components/overview/OverviewMain.jsx

```diff
@@ -1,27 +1,35 @@
 import React from 'react';
-import ImageCarousel from './sub-Components/imageCarousel/carouselMain.jsx';
+import ImageCarousel from './sub-Components/imageCarousel/Regular Carousel/carouselMain.jsx';
 import BasicDetails from './sub-Components/BasicDetails';
 import StyleSelection from './sub-Components/StyleSelection';
 import AddToBag from './sub-Components/AddToBag';
 import Description from './sub-Components/Description';
 import Axios from 'axios';
+import ImageModal from './sub-Components/ImageModal.jsx';
+import ZoomImage from './sub-Components/imageCarousel/Zoomed Carousel/ZoomImage.jsx';
+import ZoomNavigation from './sub-Components/imageCarousel/Zoomed Carousel/ZoomNavigation.jsx';
 
 class overviewMain extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      images: {
-        currentImage: undefined,
-        otherImagesInStyle: [],
-        currentThumbnailRow: undefined,
-        maximized: false
-      },
       basicDetails: {
         id: undefined,
         category: undefined,
         name: undefined,
         price: undefined
       },
+      images: {
```

Comment:
I was interested to see how you would set up zoomed vs regular images. This approach seems like the best route forward.